core/sql/common/multi-byte.h

```diff
@@ -18,12 +18,6 @@
 // under the License.
 //
 // @@@ END COPYRIGHT @@@
-/*
```

Comment:
Could you explain the provenance behind this code (and other similar files below) that allows us to remove these copyrights?